js/src/forum/components/Search.js

```diff
@@ -158,7 +151,7 @@ export default class Search extends Component {
 
         clearTimeout(search.searchTimeout);
         search.searchTimeout = setTimeout(() => {
-          if (search.searched.indexOf(query) !== -1) return;
+          if (app.cache.searched.indexOf(query) !== -1) return;
```

Comment:
This is something we want to avoid. We should be passing the searched state through a prop, not directly accessing `app.cache`. If another extension uses Search (e.g. fof/merge-discussions), they'd have to completely rewrite the config method to not use `app.cache.searched`. It should be passed as the state prop/attribute or something like that.

Reply:
Good point, I'll modify this to be `this.searched = this.props.searched || []`

Reply:
In the state? Or in the component? Do we want this available to all? 🤷

Reply:
Done